### ISSUE TITLE
Can't use dired-do-rename with ivy-mode

### ISSUE THREAD

**Issue by hmelman** [abo-abo/swiper#525]
I tried to rename a file in dired with ivy-mode enabled but I couldn't because ivy-completing-read kept matching a filename. In my case I was renaming something to remove dashes from the name, so "foo-bar.md" to "foo bar.md", so ivy still matched the original name. I could find no other workaround than to disable ivy-mode for the command and then reenable it. I think there should be an easier way, is there?

**Comment by abo-abo** [comment]
Press <kbd>C-M-j</kbd> (`ivy-immediate-done`) instead of <kbd>C-j</kbd>. It's explained in [the manual](http://oremacs.com/swiper/#key-bindings-for-single-selection-action-then-exit-minibuffer).

**Comment by hmelman** [comment]
I figured there had to be way thanks. I looked in `C-h m` at the time and didn't notice this solution, even though it's there. I do remember looking into `ivy-alt-done` and realizing it wasn't the solution, maybe the name `ivy-immediate-done` could be improved? Something like `ivy-done-using-input` or `ivy-done-current-input` or `ivy-skip-selection` or `ivy-accept-current-input`?

Also, maybe the help text could reverse the first sentence "exit with the current action, calling
it on the /current input/ instead of the current candidate." so the input vs candidate part is emphasized instead of the "exit with current action" part. So:  exit using the current /input/ (not /candidate/) calling the current action.
